library/Rules/AbstractRule.php

```diff
@@ -46,7 +46,7 @@ public function check($input)
     public function getName()
     {
         if (empty($this->name)) {
-            preg_replace('/.*\\\/', '', get_class($this));
+            return null;
         }
```

Comment:
@LanceFryar, just remove this entire block because if `$this->name` is empty return it is enough.